### ISSUE TITLE
file shared owner 

### ISSUE THREAD

**Issue by maimalash** [owncloud/android#1910]
in android how to get file owner when file share with me
in OCShare i can get file info only if i shared file

**Comment by davigonz** [comment]
Hi @maimalash , thank you for your comments. Could you please provide us more info by completing the default template while adding a new issue?

**Comment by davigonz** [comment]
Hi @maimalash , thank you for your comments. Could you please provide us more info by completing the default template while adding a new issue?

**Comment by davivel** [comment]
@davigonz, I think this is a development question. The template issue is not helpful here.

@maimalash . I can't remember from memory if this should be this way. Let me check a moment...

**Comment by davivel** [comment]
@maimalash , you are right. Currently the OC app only can get ```OCShares``` for files shared by the current user, so there is no easy way to get the file owner in the Android app. 

I'm sorry, but I'm not sure we can do anything about it in the short term.

**Comment by maimalash** [comment]
ok i add in android list that provide "shared with other"  where shared_via_users =1 
can i have list that provide file that share with me

**Comment by davivel** [comment]
Sorry, @maimalash , I don't understand your last message. Could you try to explain it again?

Thanks.

**Comment by DeepDiver1975** [comment]
🤖 Automated issue triage by Claude Code review agent.

This issue was opened before 2020 and has had no activity since then, so we are
closing it as outdated. If this is still relevant on a current ownCloud release,
please **reopen** (or open a fresh issue) with up-to-date details — thank you\!
